### ISSUE TITLE
Training workflow for DeepEdit with DL-based Image Selection

### ISSUE THREAD

**Issue by diazandr3s** [Project-MONAI/MONAILabel#31]
- Training process automatically starts when the user submits the first label. 
- Training process stops ONLY when the user clicks on the Stop training/Finish/Close System button. 
  Not absolutely relevant hyperparameters are batch size and learning rate. To start with, batch size can be set to 2, and learning rate can be set to 1e-4 and decrease by x0.1 every iteration/epoch. The system will set the learning rate to 1e-4 every time the user provides a new label. 
- The system updates the weights’ file every iteration. 
- After the user stops the training, the system keeps/saves the last model/weights and updates the YAML file or App so the user has the option to continue with the pre-trained model. 
- During the training process, a validation step is not needed. The active learning techniques will indirectly serve as a model validation process. 
  Validation is different from model quality control (QC). QC will be performed on a different dataset once the clinician/user provides X amount of labels and there is a pre-trained model. 
- The active learning techniques use the core model as inference mode. To do that, they will use the most updated model to select the next sample. 
![training_workflow](https://user-images.githubusercontent.com/11991079/116707854-086ed780-a9c7-11eb-820e-cadfb5ac1cc7.png)

**Comment by SachidanandAlle** [comment]
lets have only training and inference logic for deepedit as part of 0.1 version release... lets create a seperate feature request for next version to use deepedit for active learing..

and now 2 samples for heart/spleen will be helpful to use deepedit models..
